frontend/src/App.js

```diff
@@ -12,6 +13,6 @@ export default class App extends React.Component {
     });
   }
   render() {
-    return <TestComponent />;
```

Comment:
Don't delete this component 
I will change its name and use it as a prototype for shred components and styles like colors etc...